usecases/events/retrieve_aggregate_event_impact.py

```diff
@@ -8,15 +8,17 @@
 phq = Client(access_token=ACCESS_TOKEN)
 
 
-# The PredictHQ API offers two types of aggregation on events.
+# The PredictHQ Aggegate Event Impact API can be used to find the correlation between your demand and the events data.
+# The API offers two types of aggregation on events which you can specify in impact_rank param.
```

Comment:
I think a more precise description here is one similar to what we say in the developer documentation, i.e.,

> Use the impact_rank parameter to specify the rank to use for calculating the aggregate impact

We always aggregate on the _date_ but can use different `impact_rank` values to specify how we calculate the impact.

Reply:
Also, same suggestion on API vs endpoint.